query_detections_service: 'query_obstacle_detections'

# Fake Detector Parameters
object_position_noise: 0.5
object_velocity_noise: 0.5
object_detection_rate: 0.7
object_detection_dist: 10.0

# Obstacle position parameters
ob_start_x: [10.0, 10.0, 10]
ob_start_y: [0.0, 5.0, -5] 
ob_start_z: [10.0, 10.0, 10]

nr_of_obstacles: 2